bigbluebutton-html5/imports/api/question-quiz/server/publishers.js

```diff
@@ -0,0 +1,127 @@
+import { Meteor } from 'meteor/meteor';
+import { check } from 'meteor/check';
+import Logger from '/imports/startup/server/logger';
+import Users from '/imports/api/users';
+import QuestionQuizs from '/imports/api/question-quiz';
+import AuthTokenValidation, {
+  ValidationStates,
+} from '/imports/api/auth-token-validation';
+import { DDPServer } from 'meteor/ddp-server';
+
+Meteor.server.setPublicationStrategy('questionQuizs', DDPServer.publicationStrategies.NO_MERGE);
+
+const ROLE_MODERATOR = Meteor.settings.public.user.role_moderator;
+
+function currentQuestionQuiz(secretQuestionQuiz) {
+  check(secretQuestionQuiz, Boolean);
+  const tokenValidation = AuthTokenValidation.findOne({
+    connectionId: this.connection.id,
+  });
+  if (
+    !tokenValidation
+    || tokenValidation.validationStatus !== ValidationStates.VALIDATED
+  ) {
+    Logger.warn(
+      `Publishing QuestionQuizs was requested by unauth connection ${this.connection.id}`,
+    );
+    return QuestionQuizs.find({ meetingId: '' });
+  }
+
+  const { meetingId, userId } = tokenValidation;
+
+  const User = Users.findOne({ userId, meetingId }, { fields: { role: 1, presenter: 1 } });
+
+  if (!!User && User.presenter) {
+    Logger.debug('Publishing QuestionQuizs', { meetingId, userId });
+
+    const selector = {
+      meetingId,
```

Comment:
You should add a filter for the field `isCorrect` for users that are a responser

Reply:
Added isCorrect attribute in responses of users. Now we can check if user responded correct or not just by using this attribute we don't need to do further queries.